WordPress/Classes/ViewRelated/Blog/Site Settings/SiteSettingsViewController.m

```diff
@@ -1333,7 +1333,7 @@ - (void)showJetpackSettingsForBlog:(Blog *)blog
 
     NSParameterAssert(blog);
 
-    JetpackSecuritySettingsViewController *settings = [[JetpackSecuritySettingsViewController alloc] initWithBlog:blog];
+    JetpackSettingsViewController *settings = [[JetpackSettingsViewController alloc] initWithBlog:blog];
```

Comment:
Do we still need the 'Security' and 'Manage Connection' options on the Site Settings?

Reply:
As described, i chose to to keep them there for now. I can imagine that in the future we will remove them